lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -214,6 +214,7 @@ def add_additional_data(post, options)
         post[:metadata] = options[:metadata] if options[:metadata]
         post[:metadata] = {} if post[:metadata].nil?
         post[:metadata][:merchant_payment_code] = options[:order_id] if options[:order_id]
+        post[:processing_type] = options[:processing_type] if options[:processing_type]
```

Comment:
Out of curiosity, is this field one that they want to receive at the gateway level or is it more to flag that we need to send a specific header?

Reply:
This field is used to flag sending a specific header to the gateway.